styles/styles.css

```diff
@@ -779,3 +844,356 @@ main .section.leadership-content .columns > div > div > h6 {
 }
 
 
+main .zelta-section {
+  padding-bottom: 0;
+}
+
+main .zelta-section::after {
+  content: "";
+  z-index: 1;
+  background-image: var(--gradient-left-right);
+  display: block;
+  height: 1px;
+  margin-top: 32px;
+}
+
+main .section.accelerate-section .columns img {
```

Comment:
should images in all sections in columns behave this way or just for this section?